### PR TITLE
fix incorrect and broken links.

### PR DESCRIPTION
**Pull request by plainheart**
(1) fixed incorrect and broken links.
(2) removed some special characters in gl option.
(3) tweaked #114: replace semicolon with comma.
(4) added missing symbols.

### CODE REVIEW

**Comment by plainheart** [comment]
@susiwen8 Please help me to review, thank you!

File: en/tutorial/getting-started.md [hunk 1]
```diff
@@ -5,11 +5,11 @@
 
 First, install ECharts using one of the following methods:
 
-* Download official source release from [Apache ECharts (incubating) website](${websitePath}/download.html). Then [build](https://github.com/apache/incubator-echarts#build) from the source release.
+* Download official source release from [Apache ECharts (incubating) website](${websitePath}/en/download.html). Then [build](https://github.com/apache/incubator-echarts#build) from the source release.
 
 * Download from [GitHub](https://github.com/apache/incubator-echarts/releases)
 
-* Using npm: `npm install echarts --save`. [Using ECharts with webpack](${websitePath}/tutorial.html#Use%20ECharts%20with%20webpack)
+* Using npm: `npm install echarts --save`. [Using ECharts with webpack](tutorial.html#Use%20ECharts%20with%20webpack)
```
Comment (susiwen8) [inline en/tutorial/getting-started.md:12]: Would be better: `${websitePath}/en/tutorial.html#Use%20ECharts%20with%20webpack`

File: en/tutorial/npm-webpack.md [hunk 2]
```diff
@@ -77,4 +77,4 @@ myChart.setOption({
 
 Available modules see [https://github.com/apache/incubator-echarts/blob/master/index.js](https://github.com/apache/incubator-echarts/blob/master/index.js)
 
-The same goes for another popular packaging tools [browserify](http://browserify.org/), which will not be introduced again here. Using [rollup](https://rollupjs.org/) to make a custom build of echarts is introduced in [Create Custom Build of ECharts](http://echarts.baidu.com/tutorial.html#Create%20Custom%20Build%20of%20ECharts)
+The same goes for another popular packaging tools [browserify](http://browserify.org/), which will not be introduced again here. Using [rollup](https://rollupjs.org/) to make a custom build of echarts is introduced in [Create Custom Build of ECharts](tutorial.html#Create%20Custom%20Build%20of%20ECharts)
```
Comment (susiwen8) [inline en/tutorial/npm-webpack.md:80]: Same `${websitePath}/en/`

File: zh/tutorial/getting-started.md [hunk 3]
```diff
@@ -5,11 +5,11 @@
 
 你可以通过以下几种方式获取 ECharts。
 
-* 从 [Apache ECharts (incubating) 官网下载界面](${websitePath}/download.html) 获取官方源码包后构建。
+* 从 [Apache ECharts (incubating) 官网下载界面](${websitePath}/zh/download.html) 获取官方源码包后构建。
 
 * 在 ECharts 的 [GitHub](https://github.com/apache/incubator-echarts/releases) 获取。
 
-*  通过 npm 获取 echarts，`npm install echarts --save`，详见“[在 webpack 中使用 echarts](${websitePath}/tutorial.html#%E5%9C%A8%20webpack%20%E4%B8%AD%E4%BD%BF%E7%94%A8%20ECharts)”
+*  通过 npm 获取 echarts，`npm install echarts --save`，详见“[在 webpack 中使用 echarts](tutorial.html#%E5%9C%A8%20webpack%20%E4%B8%AD%E4%BD%BF%E7%94%A8%20ECharts)”
```
Comment (susiwen8) [inline zh/tutorial/getting-started.md:12]: `${websitePath}/zh/tutorial.html#%E5%9C%A8%20webpack%20%E4%B8%AD%E4%BD%BF%E7%94%A8%20ECharts`

File: zh/tutorial/echarts-gl.md [hunk 4]
```diff
@@ -241,24 +241,24 @@ myChart.setOption({
...
 
-这段代码中我们又在刚才的例子基础上加入了 [visualMap](http://echarts.baidu.com/option.html#visualMap) 组件，将`Life Expectancy`这一列数据映射到了不同的颜色。
+这段代码中我们又在刚才的例子基础上加入了 [visualMap](option.html#visualMap) 组件，将`Life Expectancy`这一列数据映射到了不同的颜色。
 
 除此之外我们还把原来默认的透视投影改成了正交投影。正交投影在某些场景中可以避免因为近大远小所造成的表达错误。
 
 ![](~gl/scatter3D-color.png)
 
-当然，除了 [visualMap](http://echarts.baidu.com/option.html#visualMap) 组件，还可以利用其它的 ECharts 内置组件并且充分利用这些组件的交互效果，比如 [legend](http://echarts.baidu.com/option.html#legend)。也可以像 [三维散点图和散点矩阵结合使用](http://echarts.baidu.com/examples/editor.html?c=scatter3d-scatter&gl=1) 这个例子一样实现二维和三维的系列混搭。
+当然，除了 [visualMap](option.html#visualMap) 组件，还可以利用其它的 ECharts 内置组件并且充分利用这些组件的交互效果，比如 [legend](option.html#legend)。也可以像 [三维散点图和散点矩阵结合使用](${galleryEditorPath}scatter3d-scatter&gl=1) 这个例子一样实现二维和三维的系列混搭。
```
Comment (susiwen8) [inline zh/tutorial/echarts-gl.md:250]: `${websitePath}/zh/option.html`

File: zh/tutorial/echarts-gl.md [hunk 5]
```diff
@@ -119,23 +119,23 @@ option = {
...
+    [1314, 39, 645526, "Canada", 1800],
+    [985, 32, 321675013, "China", 1800],
+    [864, 32.2, 345043, "Cuba", 1800],
+    [1244, 36.5731262, 977662, "Finland", 1800],
+    ...
 ]
 ```
 
-在 ECharts 4 中我们可以使用 dataset 组件非常方便地引入这份数据。如果对 dataset 还不熟悉的话可以看[dataset使用教程](http://echarts.baidu.com/tutorial.html#%E4%BD%BF%E7%94%A8%20dataset%20%E7%AE%A1%E7%90%86%E6%95%B0%E6%8D%AE)
+在 ECharts 4 中我们可以使用 dataset 组件非常方便地引入这份数据。如果对 dataset 还不熟悉的话可以看[dataset使用教程](tutorial.html#%E4%BD%BF%E7%94%A8%20dataset%20%E7%AE%A1%E7%90%86%E6%95%B0%E6%8D%AE)
```
Comment (susiwen8) [inline zh/tutorial/echarts-gl.md:138]: `${websitePath}/zh/tutorial.html#%E4%BD%BF%E7%94%A8%20dataset%20%E7%AE%A1%E7%90%86%E6%95%B0%E6%8D%AE`

File: zh/tutorial/echarts-gl.md [hunk 6]
```diff
@@ -98,7 +98,7 @@ for (var i = 0; i < 1000; i++) {
 
 每一项都包含了`x`, `y`, `z`三个值，这三个值会分别被映射到笛卡尔坐标系的 x 轴，y 轴和 z 轴上。
 
-然后我们可以使用 GL 提供的 [scatter3D](http://echarts.baidu.com/option-gl.html#series-scatter3D) 系列类型把这些数据画成三维空间中正态分布的点。
+然后我们可以使用 GL 提供的 [scatter3D](option-gl.html#series-scatter3D) 系列类型把这些数据画成三维空间中正态分布的点。
```
Comment (susiwen8) [inline zh/tutorial/echarts-gl.md:101]: `${websitePath}/zh/option-gl.html#series-scatter3D`

File: zh/tutorial/echarts-gl.md [hunk 7]
```diff
@@ -30,17 +30,17 @@ import 'echarts-gl';
 
 ## 声明一个基础的三维笛卡尔坐标系
 
-引入 ECharts 和 ECharts GL 后，我们先来声明一个基础的三维笛卡尔坐标系用于绘制三维的散点图，柱状图，曲面图等常见的统计图。
+引入 ECharts 和 ECharts GL 后，我们先来声明一个基础的三维笛卡尔坐标系用于绘制三维的散点图，柱状图，曲面图等常见的统计图。
 
-在 ECharts 中我们有 [grid](http://echarts.baidu.com/option.html#grid) 组件用于提供一个矩形的区域放置一个二维的笛卡尔坐标系，以及笛卡尔坐标系上上的 x 轴（[xAxis](http://echarts.baidu.com/option.html#xAxis)）和 y 轴（[yAxis](http://echarts.baidu.com/option.html#yAxis)）。对于三维的笛卡尔坐标系，我们在 GL 中提供了 [grid3D](http://echarts.baidu.com/option-gl.html#grid3D) 组件用于划分一块三维的笛卡尔空间，以及放置在这个 [grid3D](http://echarts.baidu.com/option-gl.html#grid3D) 上的 [xAxis3D](http://echarts.baidu.com/option-gl.html#xAxis3D), [yAxis3D](http://echarts.baidu.com/option-gl.html#yAxis3D), [zAxis3D](http://echarts.baidu.com/option-gl.html#zAxis3D)。
+在 ECharts 中我们有 [grid](option.html#grid) 组件用于提供一个矩形的区域放置一个二维的笛卡尔坐标系，以及笛卡尔坐标系上上的 x 轴（[xAxis](option.html#xAxis)）和 y 轴（[yAxis](option.html#yAxis)）。对于三维的笛卡尔坐标系，我们在 GL 中提供了 [grid3D](option-gl.html#grid3D) 组件用于划分一块三维的笛卡尔空间，以及放置在这个 [grid3D](option-gl.html#grid3D) 上的 [xAxis3D](option-gl.html#xAxis3D), [yAxis3D](option-gl.html#yAxis3D), [zAxis3D](option-gl.html#zAxis3D)。
```
Comment (susiwen8) [inline zh/tutorial/echarts-gl.md:35]: `${websitePath}/zh/`

File: en/tutorial/renderer.md [hunk 8]
```diff
@@ -34,8 +34,8 @@ Notice: The features of rich text, shadow and texture have not been supported ye
 ECharts uses Canvas by default. If user intends to use the SVG renderer, the module of the SVG renderer should be included in ECharts bundle.
 
 + In the [pre-build](https://www.jsdelivr.com/package/npm/echarts) of ECharts, the SVG renderer has been included in [common version](https://cdn.jsdelivr.net/npm/echarts/dist/echarts.common.min.js) and [complete version](https://cdn.jsdelivr.net/npm/echarts/dist/echarts.min.js). But not in [simple version](https://cdn.jsdelivr.net/npm/echarts/dist/echarts.simple.min.js).
-+ If [build ECharts online](http://echarts.baidu.com/builder.html), the checkbox "SVG 渲染" should be checked.
-+ If [build ECharts offline](http://echarts.baidu.com/tutorial.html#Create%20Custom%20Build%20of%20ECharts), the module of the SVG renderer should be imported:
++ If [build ECharts online](${websitePath}/en/builder.html), the checkbox "SVG Renderer" should be checked.
++ If [build ECharts offline](tutorial.html#Create%20Custom%20Build%20of%20ECharts), the module of the SVG renderer should be imported:
```
Comment (susiwen8) [inline en/tutorial/renderer.md:38]: `${websitePath}/en/`

**Comment by plainheart** [comment]
I think these are okay, no need to use websitePath. It will be handled correctly when switch the language.

**Pull request merged**